test/fixtures/cli/features/CLI.feature

```diff
@@ -4,18 +4,16 @@ Feature: Interactive CLI support
     Given I start a new shell
     When I input "./features/fixtures/node_script" interactively
     And I wait for the shell to output "Starting node script" to stdout
-    Then the current stdout line is "Repeat 5 times "
+    Then the current stdout line contains "Repeat 5 times "
```

Comment:
These steps have to use "contains" because the readline library uses ANSI escape codes to manipulate the cursor now that it thinks we're in a real TTY

i.e. this step could also be `the current stdout line is "\e[1G\e[0JRepeat 5 times \e[16G"` but that seemed worse than doing a substring match